sources/Filer/HelpTab.cpp

```diff
@@ -178,7 +178,10 @@ HelpTab::HelpTab()
 				.Add(fDocsButton)
 				.AddGlue()
 			.End()
+		.AddGlue(100.0)
 		.End();
+
+	SetExplicitMaxSize(BSize(B_SIZE_UNLIMITED, B_SIZE_UNLIMITED));
```

Comment:
AddStrut() doesn't cut it. That would only add some space below the two buttons. AddGlue() makes all the widgets stick to the upper windows border, which is the intention.